### PR TITLE
Add .import command

### LINKED ISSUE #77: Support CSV import
SQLite [allows users to import data from CSV files][1]:

```sh
$ cat foo.csv
one,two,three
1,2,3
4,5,6

$ sqlite3 foo.db
SQLite version 3.22.0 2018-01-22 18:45:57
Enter ".help" for usage hints.
sqlite> .mode csv
sqlite> .import foo.csv foo
sqlite> .schema foo
CREATE TABLE foo(
  "one" TEXT,
  "two" TEXT,
  "three" TEXT
);
sqlite> select * from foo;
1,2,3
4,5,6
sqlite>
```

However, LiteCLI gives a syntax error:

```
$ litecli foo.db
Version: 1.2.0
Mail: https://groups.google.com/forum/#!forum/litecli-users
GitHub: https://github.com/dbcli/litecli
foo.db> .mode csv
Changed table format to csv
Time: 0.000s
foo.db> .import foo.csv foo
near ".": syntax error
foo.db>
```

Aside from `.`, CSV paths may include other unexpected characters like `:` and `/`, e.g. in this example from the official documentation:

```
sqlite> .mode csv
sqlite> .import C:/work/somedata.csv tab1
```

This likely affects [CSV export][2] as well.

Can CSV import (and, ideally, output) support be added to LiteCLI?

[1]: https://sqlite.org/cli.html#csv_import
[2]: https://sqlite.org/cli.html#csv_export

### PR DESCRIPTION
**Pull request by zzl0**
## Description

Add .import command (fix #77).

## Checklist

- [x] I've added this contribution to the `changelog.md` file.

## Example

litecli_import

```bash
$ cat data.csv
t1 11
t2 22
```

```
ex1.db> .schema tbl1
+--------------------------------------------------+
| sql                                              |
+--------------------------------------------------+
| CREATE TABLE tbl1(one varchar(10), two smallint) |
+--------------------------------------------------+
Time: 0.012s

ex1.db> select * from tbl1;
0 rows in set
Time: 0.000s

ex1.db> .import ./data.csv tbl1;
Inserted 2 rows into tbl1
Time: 0.007s

ex1.db> select * from tbl1;
+-----+-----+
| one | two |
+-----+-----+
| t1  | 11  |
| t2  | 22  |
+-----+-----+
2 rows in set
Time: 0.011s
```

### CODE REVIEW

File: litecli/packages/completion_engine.py [hunk 1]
```diff
@@ -110,7 +110,7 @@ def suggest_special(text):
     if cmd in ["\\f", "\\fs", "\\fd"]:
         return [{"type": "favoritequery"}]
 
-    if cmd in ["\\d", "\\dt", "\\dt+", ".schema"]:
+    if cmd in ["\\d", "\\dt", "\\dt+", ".schema", ".import"]:
```
Comment (amjith) [inline litecli/packages/completion_engine.py]: Is this the right place for the `.import` statement? 

I would have thought that it had to be **filename** and table/view/schema.

**Comment by amjith** [comment]
Would it be possible to add a test case for this?

File: litecli/packages/completion_engine.py (hunk 1 above)
Comment (zzl0) [reply litecli/packages/completion_engine.py]: Yes, the usage of `.import` is `.import filename table`. For `filename`, I am not sure if there is a way to provide some suggestions for `filename`. I put it here, just want to provide suggestions for `table` parameter. Do you have any suggestion for this case?

**Comment by zzl0** [comment]
> Would it be possible to add a test case for this?

Yes, I think it's possible. 

I checked the existing [`test_dbspecial`](https://github.com/dbcli/litecli/blob/master/tests/test_dbspecial.py#L6) file and [previous PR](https://github.com/dbcli/litecli/pull/71/files). So I thought we normally don't test the functionality of special command.

I will add tests in next revision.

File: litecli/packages/completion_engine.py (hunk 1 above)
Comment (amjith) [reply litecli/packages/completion_engine.py]: Yes. Take a look at the `.open` special command. I believe that provides a filename suggestion.

Comment (zzl0) [reply litecli/packages/completion_engine.py]: @amjith That looks cool, thanks. I didn't notice this.

Comment (amjith) [reply litecli/packages/completion_engine.py]: Will you be updating this PR with the filename suggestion?

Comment (zzl0) [reply litecli/packages/completion_engine.py]: Yes, I will update it and also add tests for `.import` command. ETA: by end of this week.

Comment (zzl0) [reply litecli/packages/completion_engine.py]: Updated the filename suggestion.

**Comment by amjith** [comment]
:+1:

**Pull request merged**